doozer/doozerlib/distgit.py

```diff
@@ -1611,7 +1627,7 @@ def _rebase_from_directives(self, dfp):
         for i, image in enumerate(parent_images):
             # Does this image inherit from an image defined in a different group member distgit?
             if image.member is not Missing:
-                mapped_images.append(self._mapped_image_from_member(image, original_parents, count))
+                mapped_images.append(self._mapped_image_from_member(image, original_parents[i], dfp))
```

Comment:
I am slightly confused regarding the usage of  `original_parents`, `parent_images` and `image`. Seems like these were created so we don't modify and lose their original values? But they aren't a deepcopy ?
- on line 1637, image and original_parents[i] would be the same
- throughout the func original_parents and parent_images would be the same

Reply:
`parent_images` is what we have in our yaml config; `original_parents` is the result of parsing Dockerfile FROM clauses. Maybe we could rename:
- `parent_images` into `downstream_parents`
- `original_parents` into `upstream_parents`

That should be enough to clarify. Done in 6fc899c6ca15b0093c5482be4d4bd8887bcb5ed6